lib/ansible/modules/windows/win_audit_rule.ps1

```diff
@@ -0,0 +1,182 @@
+#!powershell
+
+# Copyright: (c) 2017, Noah Sparks <nsparks@outlook.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#Requires -Module Ansible.ModuleUtils.Legacy.psm1
+#Requires -Module Ansible.ModuleUtils.SID.psm1
+
+$params = Parse-Args -arguments $args -supports_check_mode $true
+$check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
+
+# these are your module parameters
+$path = Get-AnsibleParam -obj $params -name "path" -type "str" -failifempty $true -aliases "destination","dest"
+$identity_reference = Get-AnsibleParam -obj $params -name "identity_reference" -type "str" -failifempty $true
+$rights = Get-AnsibleParam -obj $params -name "rights" -type "str"
+$inheritance_flags = Get-AnsibleParam -obj $params -name "inheritance_flags" -type "str" -default "ContainerInherit,ObjectInherit" -validateset 'None','ContainerInherit','ObjectInherit'
```

Comment:
Does this combination work? Can you specify multiple values when `validateset` checks each value. Might be best to use the `list` type here as well.

Reply:
So...it seems to work when I have that as the default value, but not when I specify it as a parameter which I guess is why I never caught it.

Reply:
Hmm, doesn't look like -validateset can be used with the list type. I wonder if it would be worth updating it to check both default values and lists?